src/vue/fields/SelectField.vue

```diff
@@ -16,19 +16,20 @@
     <button
       type="button"
       class="select-field__selected"
-      :class="{'select-field__selected--focused': isExpanded}"
+      :class="{
+        'select-field__selected--focused': isExpanded,
+        'select-field__selected--padding' : label
```

Comment:
nice try but still missed, cuz now you produced another bug
the best way to achieve the goal is to assign a modifier from the outside

![image](https://user-images.githubusercontent.com/22102559/52220983-ce2a3480-28a8-11e9-842f-aa7932036583.png)